### DISCUSSION TITLE
many associations do not released

### DISCUSSION THREAD

**Discussion by linzhidong1984** [dcm4che/dcm4chee-arc-light#3121] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/29452188/113802878-092c9900-978e-11eb-9c17-5c25c33b2b9a.png)
version:5.23.1 linux
my dcm4chee server has so many associations that last a long time have not been released.
I do not know how to deal with it.
can anyone help me? thanks a lot!

**Comment by gunterze** [comment, marked as answer]
The archive is still waiting for a C-STORE RSP for a C-STORE RQ it sent to the remote AE. You may configure a

Name | Type | Description (LDAP Attribute)
-- | -- | --
Response Timeout | integer | Timeout in ms for receive of other outstanding DIMSE RSPs than C-MOVE  or C-GET RSPs; no timeout if absent (dcmResponseTimeout)

by UI Configuration -> Devices -> dcm4chee-arc -> Child Objects -> Network Connections -> host:111112 dicom -> [Network Connection Extension](https://dcm4chee-arc-cs.readthedocs.io/en/latest/networking/config/dcmNetworkConnection.html)

**Reply by linzhidong1984** [reply]
i have set the Response Timeout value,but it still has the problem!
![image](https://user-images.githubusercontent.com/29452188/114504619-bb1e0680-9c61-11eb-8c05-d529dd839ad5.png)

**Reply by gunterze** [reply]
You may also configure a Store Timeout, dependent on bandwidth and maximal object size.

**Reply by linzhidong1984** [reply]
it does not work too!
